src/main/java/dev/slimevr/config/FilteringConfig.java

```diff
@@ -0,0 +1,86 @@
+package dev.slimevr.config;
+
+import dev.slimevr.Main;
+import dev.slimevr.vr.trackers.IMUTracker;
+import dev.slimevr.vr.trackers.Tracker;
+import dev.slimevr.vr.trackers.TrackerFilters;
+import io.eiren.yaml.YamlNode;
+
+import java.util.HashMap;
+
+
+public class FilteringConfig {
+
+	private final ConfigManager configManager;
+
+	public static String CONFIG_ROOT = "filters";
+	public static float DEFAULT_INTENSITY = 0.3f;
+	public static int DEFAULT_TICK = 1;
```

Comment:
Do we need these? Are these useful to have anymore now that the default can just be on the field?

Reply:
No, y ou still need a default value because the filtering ils always reading a value